cpp/velox/substrait/SubstraitToVeloxPlan.cc

```diff
@@ -1880,8 +1882,10 @@ void SubstraitToVeloxPlanConverter::createNotEqualFilter(
   // Value < upper
   std::unique_ptr<FilterType> upperFilter;
   if constexpr (std::is_same_v<RangeType, common::BigintRange>) {
-    upperFilter = std::make_unique<common::BigintRange>(
-        getLowest<NativeType>() /*lower*/, notVariant.value<NativeType>() - 1 /*upper*/, nullAllowed);
+    if (getLowest<NativeType>() <= notVariant.value<NativeType>() - 1) {
```

Comment:
Looks we fix two issues in this PR, I think fix here also impacts tiny_int and int, should we add tests for them?

Reply:
tiny_int (bytetype) is not supported for parquet and orc. Added test for int.